__tests__/news/news.query.test.js

```diff
@@ -159,9 +307,8 @@ describe('querries', () => {
       .then(res => res)
       .catch(e => e);
 
-    expect(response.data).not.toBeNull();
-    expect(response.data.getNewsById).toBeDefined();
-    expect(response.data.getNewsById).toHaveProperty('statusCode');
-    expect(response.data.getNewsById).toHaveProperty('message');
+    expect(res.data.getNewsById).toBeDefined();
+    expect(res.data.getNewsById).toHaveProperty('statusCode');
```

Comment:
Please, check the actual value of the `message` and `statusCode`, I mean numeric value and the error message text

Reply:
fixed